src/Ozone_Monitor/data_xtrct/ush/OznMon_MkBase.sh

```diff
@@ -65,131 +62,81 @@ done
 this_file=`basename $0`
 this_dir=`dirname $0`
 
-echo "OZNMON_SUFFIX, SATYPE = $OZNMON_SUFFIX, $SATYPE"
-
-#--------------------------------------------------------------------
-# Get the area (glb/rgn) for this suffix
-#--------------------------------------------------------------------
-export GLB_AREA=${GLB_AREA:-1}
-#area=$RAD_AREA
-#echo $area, $REGIONAL_RR
 
 #------------------------------------------------------------------
 # Set environment variables.
 #-------------------------------------------------------------------
 top_parm=${this_dir}/../../parm
 
-oznmon_version=${oznmon_version:-${top_parm}/OznMon.ver}
-if [[ -s ${oznmon_version} ]]; then
-   . ${oznmon_version}
-else
-   echo "Unable to source ${oznmon_version} file"
-   exit 2
-fi
-
 oznmon_user_settings=${oznmon_user_settings:-${top_parm}/OznMon_user_settings}
-if [[ -s ${oznmon_user_settings} ]]; then
-   . ${oznmon_user_settings}
-else
+if [[ ! -e ${oznmon_user_settings} ]]; then
    echo "Unable to source ${oznmon_user_settings} file"
    exit 3
 fi
 
+. ${oznmon_user_settings}
+if [[ $? -ne 0 ]]; then
+   echo "Error detected while sourcing ${oznmon_user_settings} file"
+   exit $?
+fi
 
 oznmon_config=${oznmon_config:-${top_parm}/OznMon_config}
-if [[ -s ${oznmon_config} ]]; then
-   . ${oznmon_config}
-else
+if [[ ! -e ${oznmon_config} ]]; then
    echo "Unable to source ${oznmon_config} file"
    exit 4
 fi
 
+. ${oznmon_config}
+if [[ $? -ne 0 ]]; then
+   echo "Error detected while sourcing ${oznmon_config} file"
+   exit $?
+fi
+
 
 #-------------------------------------------------------------------
 #  Set dates
-#    BDATE is beginning date for the 30/60 day range
+#    BDATE is beginning date for the 45 day range
 #    EDATE is ending date for 30/60 day range (always use 00 cycle) 
```

Comment:
Should this also be '45 day range?'

Reply:
Indeed, the explanation of the range should be consistent.  I'll make it so.